source/cl/test/UnitCL/include/Common.h

```diff
@@ -311,6 +311,11 @@ bool hasHalfSupport(cl_device_id device);
 /// @return True if the device has double support, false otherwise
 bool hasDoubleSupport(cl_device_id device);
 
+/// @brief Check if the device at index 'device' has double support
+/// @param[in] device - The index of the device in getDevices()
+/// @return True if the device has double support, false otherwise
```

Comment:
```suggestion
/// @return True if the device has double support, false otherwise
```
```suggestion
/// @return True if the device has Atomic64 support, false otherwise
```